instance/static/html/instance/details.html

```diff
@@ -0,0 +1,137 @@
+          <th>Value</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr><td>ID</td><td>{{instance.id}}</td></tr>
+        <tr><td>Name</td><td>{{instance.name}}</td></tr>
+        <tr><td>Created</td><td>{{instance.created | date:'yyyy-MM-dd HH:mm:ssZ' }}</td></tr>
+        <tr><td>Modified</td><td>{{instance.modified | date:'yyyy-MM-dd HH:mm:ssZ' }}</td></tr>
+        <tr><td>Type</td><td>{{instance.instance_type}}</td></tr>
+        <!-- The following are all specific to Open edX instances: -->
+        <tr><td>Domain</td><td>{{instance.domain}}<br>Sub domain: {{instance.sub_domain}}, base domain: {{instance.base_domain}}</td></tr>
+        <tr><td>LMS URL</td><td><a href="{{instance.url}}">{{instance.url}}</a></td></tr>
+        <tr><td>Studio URL</td><td><a href="{{instance.studio_url}}">{{instance.studio_url}}</a></td></tr>
+        <tr><td>Email</td><td>{{instance.email}}</td></tr>
+        <tr><td>Protocol</td><td>{{instance.protocol}}</td></tr>
+        <tr><td>Use ephemeral databases</td><td>{{instance.use_ephemeral_databases}}</td></tr>
+        <tr><td>GitHub admin org</td><td>{{instance.github_admin_organization_name}}</td></tr>
+        <tr><td>configuration_source_repo_url</td><td>{{instance.configuration_source_repo_url}}</td></tr>
+        <tr><td>configuration_version</td><td>{{instance.configuration_version}}</td></tr>
+        <tr><td>edx_platform_repository_url</td><td>{{instance.edx_platform_repository_url}}</td></tr>
+        <tr><td>edx_platform_commit</td><td>{{instance.edx_platform_commit}}</td></tr>
+        <tr><td>openedx_release</td><td>{{instance.openedx_release}}</td></tr>
+        <tr><td>configuration_extra_settings</td><td><pre>{{instance.configuration_extra_settings}}</pre></td></tr>
+      </tbody>
+    </table>
+
+    <h4>Status</h4>
+    <table>
+      <tbody>
+        <tr><td>MySQL provisioned</td><td>{{instance.mysql_provisioned}}</td></tr>
+        <tr><td>Mongo provisioned</td><td>{{instance.mongo_provisioned}}</td></tr>
+        <tr><td>Swift provisioned</td><td>{{instance.swift_provisioned}}</td></tr>
+      </tbody>
+    </table>
+  </tab>
+  <tab heading="Log" active="instance_active_tabs.log_tab">
+    <p>This log does not include events from each App Server or VM. Select an App Server on the "App Servers" tab to view those logs.</p>
+    <pre class="instance-log" ng-repeat="line in instance.log_entries track by $index">{{ line.created | date:'yyyy-MM-dd HH:mm:ssZ' }} | {{ line.level }} | {{ line.text }}</pre>
+  </tab>
+  <tab heading="Pull Request (#{{pr.github_pr_number}})" ng-if="instance.source_pr" ng-init="pr = instance.source_pr" active="instance_active_tabs.pr_tab">
+    <h4>Pull Request Details</h4>
+    <p>This instance was automatically generated for the GitHub pull request <a href="{{pr.github_pr_url}}">{{pr.fork_name}} #{{pr.github_pr_number}} ({{pr.branch_name}})</a></p>
+
+    <p>If the PR has changed, you can update the instance settings based on the PR:</p>
```

Comment:
@bradenmacdonald It might be worth mentioning here that updating the settings won't spawn a new app server.